conftest.py

```diff
@@ -664,7 +664,17 @@ def setup_test_colo(
         """Setup things needed for setting up the colo pinning tests"""
         # get test setup
         test_dir = fileutils.make_test_dir(level=2)
-        sr_test_script = fileutils.get_test_conf_path("send_data_local_smartredis.py")
+
+        # Jpnote
+        if db_args["db_identifier"]:
+            sr_test_script = fileutils.get_test_conf_path(
+                "send_data_local_smartredis_with_dbid.py"
+            )
+            print("yes", db_args["db_identifier"])
+        else:
+            sr_test_script = fileutils.get_test_conf_path(
+                "send_data_local_smartredis.py"
```

Comment:
It would be better to put the python filename into a temp var and then call fileutils.get_test_conf_path() on the temp var outside the conditional -- this eliminates some code duplication